api/projects/migrations/0022_rename_identity_overrides_migration_status_project_identity_overrides_v2_migration_status.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.23 on 2023-12-11 15:14
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0021_add_identity_overrides_migration_status'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='project',
+            old_name='identity_overrides_migration_status',
+            new_name='identity_overrides_v2_migration_status',
+        ),
+    ]
```

Comment:
I'm not entirely sure this is worth doing at all but, if you think it is, I'd rather just update the old migration and fix the issues in the staging database.

Reply:
Done, but I'll probably ask for your assistance on reapplying the migration.